ui/Makefile

```diff
@@ -11,7 +11,8 @@ SOURCES += $(shell find apps -type d \( -name node_modules \) -prune -o -print)
 # tracking package.json files of monorepo packages is the easiest way to discover such situations
 PACKAGE_JSON_FILES := $(shell find . -type d \( -name node_modules \) -prune -false -o -name package.json)
 
-export REACT_APP_ROX_PRODUCT_BRANDING := $(shell $(MAKE) --quiet --no-print-directory -C .. product-branding)
+ROX_PRODUCT_BRANDING ?= $(shell $(MAKE) --quiet --no-print-directory -C .. product-branding)
+export REACT_APP_ROX_PRODUCT_BRANDING := $(ROX_PRODUCT_BRANDING)
```

Comment:
@msugakov What's the purpose for this change?

Reply:
WOW. This PR is a mess! I wonder how did you discover it, do you check all PRs in the repo?
I'll close the PR and hope to be notified (through `rhtap-maintainers`) when new ones like it get opened. That would mean I have a misconfiguration somewhere.

The proper change is here https://github.com/stackrox/stackrox/pull/8689/files#r1416228188